src/js/components/Values/IssuesByBallotItemDisplayList.jsx

```diff
@@ -72,6 +72,14 @@ class IssuesByBallotItemDisplayList extends Component {
   }
 
   render () {
+    const handleEnterHoverLocalArea = () => {
+      this.props.handleLeaveCandidateCard();
```

Comment:
Since this function is coming in as a prop, and may or may not exist, you want to wrap it with:
```
if (this.props.handleLeaveCandidateCard) {
  this.props.handleLeaveCandidateCard();
}
```